src/libraries/System.Drawing.Common/tests/BitmapTests.cs

```diff
@@ -1750,22 +1750,53 @@ public void FromNonSeekableStream()
             }
         }
 
-        private class NonSeekableStream : Stream
+        [ConditionalTheory(Helpers.IsDrawingSupported)]
+        [InlineData(false, true)]
+        [InlineData(true, true)]
+        [InlineData(true, false)]
+        public void SaveToRestrictiveStream(bool disableRead, bool disableSeek)
```

Comment:
Nit: rather than double-negatives, how about canRead and canSeek?

Reply:
I actually spent some time considering the name here. I felt that Can* was misleading since a value of true doesn’t actually vary tee the stream turns true: it just means the stream delegates to the backing stream.

Reply:
I guess I could remove double negatives and just comment the meaning of Can* is limited to context of wrapping stream.